From 416ac2f37225fefa56ac1749709ca54f92b1f8cb Mon Sep 17 00:00:00 2001
From: Ziad Elhanafy <ziad.elhanafy@arm.com>
Date: Mon, 20 Nov 2023 13:27:59 +0000
Subject: [PATCH] kronos: Re-add test_s003 to Secure Storage tests

test_s003 was previously removed as it saturates
the RSS storage, this caused the linux boot to fail
when TF-A attempts to use the RSS PSA service as it
can not find any more storage.

Re-add this test but make it the final one to give
the Primary Compute the time to finish the service
request from RSS before reaching this test.

Upstream-Status: Inappropriate [Kronos-specific]
Signed-off-by: Ziad Elhanafy <ziad.elhanafy@arm.com>
---
 api-tests/dev_apis/storage/its_1.0-beta2_testsuite.db | 1 +
 api-tests/dev_apis/storage/its_1.0_testsuite.db       | 1 +
 api-tests/dev_apis/storage/its_testsuite.db           | 1 +
 api-tests/dev_apis/storage/ps_1.0-beta2_testsuite.db  | 1 +
 api-tests/dev_apis/storage/ps_1.0_testsuite.db        | 1 +
 api-tests/dev_apis/storage/ps_testsuite.db            | 1 +
 6 files changed, 6 insertions(+)

diff --git a/api-tests/dev_apis/storage/its_1.0-beta2_testsuite.db b/api-tests/dev_apis/storage/its_1.0-beta2_testsuite.db
index 2cd7968..1459723 100644
--- a/api-tests/dev_apis/storage/its_1.0-beta2_testsuite.db
+++ b/api-tests/dev_apis/storage/its_1.0-beta2_testsuite.db
@@ -29,5 +29,6 @@ test_s007
 test_s008
 test_s009
 test_s010
+test_s003
 
 (END)
diff --git a/api-tests/dev_apis/storage/its_1.0_testsuite.db b/api-tests/dev_apis/storage/its_1.0_testsuite.db
index 2cd7968..1459723 100644
--- a/api-tests/dev_apis/storage/its_1.0_testsuite.db
+++ b/api-tests/dev_apis/storage/its_1.0_testsuite.db
@@ -29,5 +29,6 @@ test_s007
 test_s008
 test_s009
 test_s010
+test_s003
 
 (END)
diff --git a/api-tests/dev_apis/storage/its_testsuite.db b/api-tests/dev_apis/storage/its_testsuite.db
index 1d67e22..eff3b67 100644
--- a/api-tests/dev_apis/storage/its_testsuite.db
+++ b/api-tests/dev_apis/storage/its_testsuite.db
@@ -29,5 +29,6 @@ test_s007
 test_s008
 test_s009
 test_s010
+test_s003
 
 (END)
diff --git a/api-tests/dev_apis/storage/ps_1.0-beta2_testsuite.db b/api-tests/dev_apis/storage/ps_1.0-beta2_testsuite.db
index 47436d4..47c4843 100644
--- a/api-tests/dev_apis/storage/ps_1.0-beta2_testsuite.db
+++ b/api-tests/dev_apis/storage/ps_1.0-beta2_testsuite.db
@@ -36,5 +36,6 @@ test_s014
 test_s015
 test_s016
 test_s017
+test_s003
 
 (END)
diff --git a/api-tests/dev_apis/storage/ps_1.0_testsuite.db b/api-tests/dev_apis/storage/ps_1.0_testsuite.db
index 47436d4..47c4843 100644
--- a/api-tests/dev_apis/storage/ps_1.0_testsuite.db
+++ b/api-tests/dev_apis/storage/ps_1.0_testsuite.db
@@ -36,5 +36,6 @@ test_s014
 test_s015
 test_s016
 test_s017
+test_s003
 
 (END)
diff --git a/api-tests/dev_apis/storage/ps_testsuite.db b/api-tests/dev_apis/storage/ps_testsuite.db
index 24da4b3..1f6cef2 100644
--- a/api-tests/dev_apis/storage/ps_testsuite.db
+++ b/api-tests/dev_apis/storage/ps_testsuite.db
@@ -36,5 +36,6 @@ test_s014
 test_s015
 test_s016
 test_s017
+test_s003
 
 (END)
-- 
2.25.1

